internal/sbi/api_uepolicy.go

```diff
@@ -0,0 +1,57 @@
+/*
+ * Npcf_UEPolicyControl
+ *
+ * UE Policy Control Service API
+ *
+ * API version: 1.0.0
+ * Generated by: OpenAPI Generator (https://openapi-generator.tech)
+ */
+
+package sbi
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+func (s *Server) getUePolicyRoutes() []Route {
+	return []Route{
+		{
+			Method:  http.MethodDelete,
+			Pattern: "/policies/{polAssoId}",
+			APIFunc: s.PoliciesPolAssoIdDelete,
+		},
+		{
+			Method:  http.MethodGet,
+			Pattern: "/policies/{polAssoId}",
+			APIFunc: s.PoliciesPolAssoIdGet,
+		},
+		{
+			Method:  http.MethodPost,
+			Pattern: "/policies/{polAssoId}/update",
+			APIFunc: s.PoliciesPolAssoIdUpdatePost,
+		},
+		{
+			Method:  http.MethodPost,
+			Pattern: "/policies",
+			APIFunc: s.PoliciesPost,
+		},
+	}
+}
+
+// PoliciesPolAssoIdDelete -
+func (s *Server) PoliciesPolAssoIdDelete(c *gin.Context) {
+}
+
+// PoliciesPolAssoIdGet -
+func (s *Server) PoliciesPolAssoIdGet(c *gin.Context) {
+}
+
+// PoliciesPolAssoIdUpdatePost -
+func (s *Server) PoliciesPolAssoIdUpdatePost(c *gin.Context) {
+}
+
+// PoliciesPost -
+func (s *Server) PoliciesPost(c *gin.Context) {
+}
```

Comment:
Should set status to `501 Not Implemented`. By default, the gin returns `200 OK`. 

```suggestion
func (s *Server) PoliciesPolAssoIdDelete(c *gin.Context) {
  c.JSON(501, /* problem details */)
}

// PoliciesPolAssoIdGet -
func (s *Server) PoliciesPolAssoIdGet(c *gin.Context) {
  c.JSON(501, /* problem details */)
}

// PoliciesPolAssoIdUpdatePost -
func (s *Server) PoliciesPolAssoIdUpdatePost(c *gin.Context) {
  c.JSON(501, /* problem details */)
}

// PoliciesPost -
func (s *Server) PoliciesPost(c *gin.Context) {
  c.JSON(501, /* problem details */)
}
```

Reply:
@yccodr Then I define an new error for `501 Not Implemented` in pcf_util.go? Or just empty body is fine.

Reply:
Just define in this file. 
You can use recommended code with nil body,